lte/gateway/python/magma/pipelined/main.py

```diff
@@ -131,6 +131,11 @@ def on_exit_server_thread():
         start_check_quota_server(run_flask, bridge_ip, no_quota_port, False,
                                  on_exit_server_thread)
 
+
+    # start debug server
+    debug_server_port = service_config.get('debug_server_port', 9999)
```

Comment:
I'd rather we fail the debug server start if the port is not present, will avoid some headache in the future

Reply:
Also we should make this optional, there is a security risk in running eval on arbitrary command, I suppose the only thing that redirects to localhost is from our redirection flows but those are to a different port

Reply:
As long as only authorized users can access the gateway, others shouldn't be able to reach this port. if port is a concern I can change this to a local unix domain port.  I don't think this should be optional, that defeats the purpose of the debug server. We should enable it by default so that in cases where we are screwed with hard to debug issues we can introspect the service.

Reply:
Sure I agree we should probably always run this, but I'm not a fan of specifying the 9999 port just like that. Maybe lets just make this a required yml arg, without a default value?

Reply:
Changed this to pipelined domain socket. We shouldn't have collisions here. Let me know if you had additional feedback here. Thanks